cronjobparser.py

```diff
@@ -18,6 +18,29 @@ def __init__(self, command, minute, user="root", hour="*", dom="*",
         self.month = month
         self.dow = dow
 
+    @classmethod
+    def from_time(cls, command, user, time):
```

Comment:
To my understanding this is an alternative __init__ function. Why would we need this?
Should we add an optional "time" parameter to the init function?
I might be confusing, that sometimes the object is created using

    obj = CronJob()

and sometimes via

   obj = CronJob.from_time()

Reply:
Exactly, this is pretty much an additional constructor.

I added this because there were several lines like
```python
obj = CronJob(command, date_fragment[0], user, date_fragment[1], date_fragment[2], date_fragment[3], date_fragment[4])
```
in the appliance. They are now simplified to
```python
obj = CronJob.from_time(command, user, date_fragment)
```

But I wouldn't like to add another ``time`` parameter to ``__init__``, because that makes the function signature of ``__init__`` confusing: Then, we could *either* specify ``minute``, ``hour``, ``dom``, ``month``, ``dow`` *or* ``time``. I think having a separate constructor is much clearer here.

Reply:
ok. thx.